### ISSUE TITLE
wiki and encryption

### ISSUE THREAD

**Issue by k2patel** [buserror/touchstream#1]
Hi Michael,

i like the overall tool, i was wondering if there is any documentation for compile time options and configuration options which can be used.
As i was looking to see if i can encrypt the data between two computer.

Please advise.

**Comment by buserror** [comment]
No, right now there isn't support for encryption -- my 'use case' is between two machines on the same desk, so I'd assume the traffic would be filtered by the switch so it's not easily sniffed from the outside. if either of the machines are compromised, I don't think any amount of encryption would help...

If I find a way to add encryption without adding latency, I'd probably jump on this tho..
